src/lib/render-lit-html.ts

```diff
@@ -112,6 +112,13 @@ declare global {
   }
 }
 
+const setAttributeToElement = (element: LitElement, name: string,
```

Comment:
huh... should this move into the ElementRenderer interface?

Reply:
Sure.